src/styles/header.scss

```diff
@@ -0,0 +1,85 @@
+// #region header
+.header {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.navi-bar {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.navi-menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.navi-item {
+  margin: 0;
+  padding-left: 20px;
+}
+
+.navi-item:first-child {
+  padding: 0;
+}
+
+.navi-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  color: #060b35;
+  height: 100%;
+  line-height: 60px;
+  padding: 0;
+  text-align: left;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.navi-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  display: flex;
+
+  text-decoration: none;
+  color: #00acdc;
+  line-height: 60px;
+  padding: 0;
+  text-align: left;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  margin-left: 0;
+}
```

Comment:
The `.is-active` class seems to repeat a lot of the styles already defined in the `.navi-link` class. To keep your CSS DRY (Don't Repeat Yourself), you could simplify this by only specifying the styles that change when the link is active. Also, don't forget to add a transition for a smooth color change.